packages/itwinui-css/src/icon/mixins.scss

```diff
@@ -3,50 +3,45 @@
 @import '../style/index';
 
 @mixin iui-icons-default {
-  @include iui-icons-color();
-  display: inline-flex;
   width: $iui-icons-default;
   height: $iui-icons-default;
 }
 
 @mixin iui-icons-small {
-  @include iui-icons-color();
-  display: inline-flex;
   width: $iui-icons-small;
   height: $iui-icons-small;
 }
 
 @mixin iui-icons-large {
-  @include iui-icons-color();
-  display: inline-flex;
   width: $iui-icons-large;
   height: $iui-icons-large;
 }
 
 @mixin iui-icons-xl {
-  @include iui-icons-color();
-  display: inline-flex;
   width: $iui-icons-xl;
   height: $iui-icons-xl;
 }
 
 @mixin iui-icons-xxl {
-  @include iui-icons-color();
-  display: inline-flex;
   width: $iui-icons-xxl;
   height: $iui-icons-xxl;
 }
 
 @mixin iui-icons-3xl {
-  @include iui-icons-color();
-  display: inline-flex;
   width: $iui-icons-3xl;
   height: $iui-icons-3xl;
 }
 
+@mixin iui-icons-display {
+  display: inline-flex;
+  flex-shrink: 0;
```

Comment:
This mixin can't be used for every icon. It causes a lot of bad changes. Not every icon/element is currently using `display: inline-flex`, and adding this mixin to every icon breaks a lot of test images.

I separated `display: inline-flex` and `flex-shrink: 0` so that the icon size mixins can be used in more places. Putting them together again means that I can't do that.